src/languages/twig.js

```diff
@@ -98,13 +135,18 @@ export default function(hljs) {
     "filter",
     "flush",
     "for",
+    "form_theme",
     "from",
     "if",
     "import",
     "include",
     "macro",
     "sandbox",
     "set",
+    "stopwatch",
+    "trans",
+    "trans_default_domain",
+    "transchoice",
```

Comment:
What does it mean for something to be both a tag name and a filter?  Referring here to lines 147/149 vs 116/117.

Reply:
Of you're just copying the lists, hmmmm.

Reply:
In Twig, if something can be used both as filter and a tag, it'd look like this,

```twig
{% trans %}
  I have some text here that I'm writing manually
{% endtrans %}
```

and

```twig
{{ some_variable | trans }}

^ shorthand for:

{% trans %}{{ some_variable }}{% endtrans %}
```